client/my-sites/pages/main.jsx

```diff
@@ -68,6 +72,7 @@ module.exports = React.createClass( {
 		};
 		return (
 			<Main classname="pages">
+				<QuerySites allSites={ true } />
```

Comment:
Thanks @obenland but we'll want to hold off on this PR until we tackle all selected site usages. 

If you look at https://github.com/Automattic/wp-calypso/projects/3 we haven't found a good way of avoiding data duplication between sites list and redux yet.